doc/man3/OPENSSL_s390xcap.pod

```diff
@@ -52,7 +54,7 @@ instruction is set to the specified 192-bit mask.
 =back
 
 The 64-bit masks are specified in hexadecimal notation. The 0x prefix is
-optional. Prefix a mask with a tilde (~) to denote a bitwise NOT operation.
+optional. Prefix a mask with a tilde, B<~>, to denote a bitwise NOT operation.
```

Comment:
`C<`

Reply:
Like I said elsewhere, will-o-the-wisp.  Changed.